x-pack/plugins/enterprise_search/common/connectors/native_connectors.ts

```diff
@@ -1652,6 +1652,32 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
         validations: [],
         value: false,
       },
+      fetch_users_by_site: {
+        default_value: false,
+        depends_on: [{ field: 'use_document_level_security', value: true }],
+        display: DisplayType.TOGGLE,
+        label: i18n.translate(
+          'xpack.enterpriseSearch.nativeConnectors.sharepoint_online.configuration.fetchUsersBySiteLabel',
+          {
+            defaultMessage: 'Fetch ACLs for users based on site membership',
```

Comment:
suggestion: I don't think "ACLs" is a well-known acronym, we should write this out fully or clarify this in some other way. Maybe "Sync user access control by site membership"?

Reply:
updated